tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/TimestampedTagValue'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag not found
+        default:
+          $ref: '#/responses/Error'
+    put:
+      tags:
+        - tags
+      summary: Update the current value of a tag
+      description: Updates the current value of the tag at the path.
+      operationId: UpdateTagCurrentValue
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+        - in: body
+          name: tagValue
+          description: Timestamped value
+          schema:
+            $ref: '#/definitions/TimestampedTagValue'
+          required: true
+      responses:
+        200:
+          description: Tag Value Updated
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: Conflict
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/values/current/timestamp:
+    get:
+      tags:
+        - tags
+      summary: Get the timestamp of a tag's current value
+      description: Returns the timestamp of the tag's current value
+      operationId: GetTagCurrentValueTimestamp
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Timestamp
+        204:
+          description: No timestamped value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/values/current/value:
+    get:
+      tags:
+        - tags
+      summary: Get the current value of a tag without its timestamp
+      description: Returns the current value of a tag
+      operationId: GetTagCurrentValue
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/TagValue'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/values/aggregates/min:
+    get:
+      tags:
+        - tagAggregates
+      summary: Get the value of a tag's "min" aggregate
+      description: Get the value of a tag's "min" aggregate
+      operationId: GetTagMinimum
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Minimum aggregate value
+          schema:
+            type: string
```

Comment:
I'll address this in another PR, as this review is already too large. The v1 route for querying subscription values needs to return numeric aggregates, instead of strings.